### DISCUSSION TITLE
Using ImageMagick to web scrape colours on a live webpage

### DISCUSSION THREAD

**Discussion by Wakojako** [ImageMagick/ImageMagick#4815] [Help]
Hey all, I am planning to build a bot that recognizes colours on a live webpage (tradingview) (see example attached below). I have experimented with colour recognition tools in Python on .jpg files but I haven't had much luck with what I am after. I have been directed to ImageMagick as a potential resource to help me. I would greatly appreciate if I could be pointed in the right direction to any resources.

![tempsnip](https://user-images.githubusercontent.com/68539515/152670696-3351d853-0c79-4b5e-92ac-accfda28f148.png)

**Comment by fmw42** [comment]
I think we will need more information about what you need to do.
